demo-app/src/main/java/cash/z/ecc/android/sdk/demoapp/Constants.kt

```diff
@@ -10,6 +10,8 @@ val ANDROID_STATE_FLOW_TIMEOUT = 5.seconds
  */
 const val MINIMAL_WEIGHT = 0.0001f
 
-// TODO [#1644]: Refactor Account ZIP32 index across SDK
-// TODO [#1644]: https://github.com/Electric-Coin-Company/zcash-android-wallet-sdk/issues/1644
+/**
+ * Until we support full multi-account feature in Demo app we use this constant as a single source of truth for
+ * account selection
+ */
```

Comment:
This should no longer be necessary? Is it even still possible to query by ZIP 32 account index?

Reply:
Never mind, saw that this is the demo app.

Reply:
This constant is part of Demo app only, and we use it only for account 0 selection across the Demo app's use cases. whereas Zashi works with `getAccounts`.